features/step_definitions/home_beta_page_steps.rb

```diff
@@ -8,4 +8,13 @@
 Then(/^I should be presented with popular tools and calculators$/) do
   expect(home_beta_page.popular_tools)
     .to have_content(I18n.t('home.show.tools_heading'))
-end
\ No newline at end of file
+end
+
+Then(/^I should be presented with the seasonal spotlight$/) do
+  save_and_open_page
```

Comment:
Remove this line as no longer is necessary.

Reply:
Good spot!! apologies

Reply:
Also tidied up the commit history